Include <ctype.h> for toupper, isspace.  This avoids implicit function
declarations and should improve compatibility with future compilers
which default to stricter C99 mode.

diff --git a/src/arg_int.c b/src/arg_int.c
index 29c20e55cccb53ee..7145bb31c5dd8f90 100644
--- a/src/arg_int.c
+++ b/src/arg_int.c
@@ -31,6 +31,8 @@ USA.
 #include "argtable2.h"
 #include <limits.h>
 
+#include <ctype.h>
+
 /* local error codes */
 enum {EMINCOUNT=1,EMAXCOUNT,EBADINT,EOVERFLOW};
 
